coalib/bears/Bear.py

```diff
@@ -294,29 +307,16 @@ def missing_dependencies(cls, lst):
 
         :param lst: A list of all already resolved bear classes (not
                     instances).
-        :return:    A list of missing dependencies.
+        :return:    A set of missing dependencies.
         """
-        dep_classes = cls.get_dependencies()
+        dep_classes = cls.BEAR_DEPS.copy()
 
         for item in lst:
             if item in dep_classes:
                 dep_classes.remove(item)
```

Comment:
the set has a builtin operation to do this, also sets are immutable: just `return set(cls.BEAR_DEPS) - set(lst)` or something like this?